### ISSUE TITLE
Release request

### ISSUE THREAD

**Issue by paulo-ferraz-oliveira** [inaka/apns4erl#241]
Would it be possible to tag a release, here and in Hex.pm?

Many thanks.

**Note**: I'm testing this with OTP 24, and may have found some issues.

- [x] wait on https://github.com/inaka/apns4erl/pull/249 being merged
- [ ] wait on https://github.com/inaka/apns4erl/pull/246 being merged
- [x] make sure the imported `gun` version doesn't mention `erlang:get_stacktrace`

**Comment by Neustradamus** [comment]
To follow

**Comment by paulo-ferraz-oliveira** [comment]
Hi. This issue is no longer relevant to me. I'll close it. Feel free to re-open or eventually tag a release. Thanks.
